experiments/rtp-hdrext/transport-wide-cc-02/index.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: transport-wide-cc-02
+permalink: /experiments/rtp-hdrext/transport-wide-cc-02/
+---
+
+This RTP header extension is an extended version of the extension defined in
+<https://tools.ietf.org/html/draft-holmer-rmcat-transport-wide-cc-extensions-01> 
```

Comment:
Are you going to submit version -02 of the internet-draft also? I'd recommend doing that before starting to use (or document) the extension.
Submitting I-Ds is free (as long as you can get the format right).

Reply:
There are no plans to submit a new internet-draft at the moment. The ambition is to have a valid URI for Chromium M74 that branches on March 7.